pegasus/sites.v3/code.org/views/promote_tools.haml

```diff
@@ -1,4 +1,5 @@
--DB[:cdo_promote_tools].all.each do |value|
+- section_name ||= 'local_tools'
```

Comment:
Yeah public/advocacy/index.md had a reference to it. I could fix that one, but I thought it was a good fallback just in case?